apis/alloydb/v1alpha1/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +kcc:proto=google.cloud.alloydb.v1beta
```

Comment:
The go tag should use v1alpha1

Reply:
I believe this tag matches the underlying proto version of the GCP resource. In this case, it should be v1beta for both v1alpha1 and v1beta1 AlloyDBCluster.

Reply:
You are right. I miss read the line.